ui/infra-compare/InfraCompare.jsx

```diff
@@ -64,26 +64,28 @@ class InfraCompareView extends React.PureComponent {
   getDisplayResults = (origResultsMap, newResultsMap, tableNames) => {
     let compareResults = new Map();
     tableNames.forEach((jobName) => {
-      jobName = jobName.replace(/-\d+$/, '');
-      const oldResults = origResultsMap.filter(
+      const originalResults = origResultsMap.filter(
         (job) => job.job_type__name.replace(/-\d+$/, '') === jobName,
```

Comment:
jobName is not split, so will the .replace(/-\d+$/, '') result in not matching any jobs ending in -X ?